### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by not-nef** [tukaan/tukaan#51] [Question] [answered]
@rdbende pls no

**Comment by rdbende** [comment, marked as answer]
No, it's not.

**Comment by not-nef** [comment]
Thank you this is awesome

**Comment by Moosems** [comment]
Nah man. This is pretty awesome

**Comment by whatamithinking** [comment]
This looks really really promising and could finally allow me to escape my current GUI platform, Ignition, for something open source.

Are you sure this isn't dead? 
Based on commit history and the todo list it seems like it may be a bit of a zombie

**Reply by rdbende** [reply]
See #148
